janetnim/maintenance-tracker
# #157896031 Design Profile Page

## **What does this PR do?**
Create a profile form for user and admin to view their personal details

## **Description of Task to be completed?**
1. a profile page should be designed with a form with personal details
2. add edit and save changes buttons

## **How should this be manually tested?**
After adding the html and css files to the UI directory, open the profile file in your browser to see if all components of the page appear and work as they should

## ** What are the relevant pivotal tracker stories?**
#157896031

![image](https://user-images.githubusercontent.com/29487740/40589914-9b92a988-61fe-11e8-8d98-54bb1a59cf55.png)
